core/src/trezor/ui/__init__.py

```diff
@@ -124,7 +124,10 @@ def backlight_fade(val: int, delay: int = 14000, step: int = 15) -> None:
             display.backlight(val)
             return
     current = display.backlight()
-    if current > val:
+    if current < 0:
```

Comment:
This exists because of unit tests:
```
class TestSignSegwitTxNativeP2WPKH
  test_send_native_p2wpkh ... errored: Value must be between 0 and 255
Traceback (most recent call last):
  File "/home/mmilata/work/trezor-firmware-alt/core/tests/unittest.py", line 237, in run_class
  File "test_apps.bitcoin.segwit.signtx.native_p2wpkh.py", line 167, in test_send_native_p2wpkh
  File "../src/apps/bitcoin/sign_tx/bitcoin.py", line 54, in signer
  File "../src/apps/bitcoin/sign_tx/progress.py", line 34, in init
  File "../src/apps/bitcoin/sign_tx/progress.py", line 118, in report_init
  File "../src/trezor/ui/layouts/tt_v2/__init__.py", line 1127, in bitcoin_progress
  File "../src/trezor/ui/layouts/tt_v2/__init__.py", line 1100, in __init__
  File "../src/trezor/ui/__init__.py", line 130, in backlight_fade
ValueError: Value must be between 0 and 255
```

Eventually we should move all backlight handling to rust so there's no need to use it from python directly.